var/spack/repos/builtin/packages/py-matplotlib/package.py

```diff
@@ -189,6 +189,8 @@ def configure(self):
                 setup.write('[libs]\n')
                 setup.write('system_freetype = True\n')
                 setup.write('system_qhull = True\n')
+            if self.spec.satisfies('%clang'):
+                setup.write('enable_lto = False\n')
```

Comment:
Can you indent this to be inside of the first if-statement? It looks like this option is only available in 3.3+

Reply:
Good idea, I fixed it.  I'll try building an older version of matplotlib to make sure there isn't some other problem with Spack-built versions of Clang.

Reply:
py-matplotlib@3.2.2%clang@12 built just fine without any changes to `setup.cfg`.